src/components/TicketListDefaultValueWrapper.tsx

```diff
@@ -0,0 +1,82 @@
+import { ReactNode, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { useQuery } from '@tanstack/react-query';
+import { fetchOrganizations } from '../api/cristinApi';
+import { fetchUser, fetchUsers } from '../api/roleApi';
+import { TicketSearchParam } from '../api/searchApi';
+import { RootState } from '../redux/store';
+import { Organization } from '../types/organization.types';
+import { RoleName } from '../types/user.types';
+import { TicketStatus } from '../types/publication_types/ticket.types';
+
+interface TicketListDefaultValueWrapperProps {
+  children: ReactNode;
+}
```

Comment:
Vet ikke helt hva som er enklest her her, så dette er bare en btw for læringens del, men det kan jo være en bedre løsning også:
I stedet for at dette er en egen komponent, kunne det vært en hook, `useSetDefaultTicketSearchParams()` elns, som bare setter params uten å ha noe med selve rendring å gjøre. Se en av de som ligger i `utils/hooks` for litt hvordan en hook kan se ut.

Reply:
Interessant!

Reply:
Prøver denne